containers/Dockerfile.f36

```diff
@@ -0,0 +1,18 @@
+FROM quay.io/fedora/fedora:36-x86_64
+ 
+ENV XDG_CACHE_HOME=/root/.cache
+RUN dnf update -y && dnf install -y iproute rsync git traceroute unzip bzip2 \
+  bzip2-devel libffi-devel sqlite-devel sqlite-devel rpm-sign expect \
+  make curl wget tar procps-ng which sudo unzip findutils grep ncurses-devel \
+  openssl-devel zlib-devel krb5-devel make cmake gcc gcc-c++ rpm-build \
+  python3.6 python3.7 python3.8 python3.9 python3.10 python3.11 python3-dnf \
```

Comment:
python3.6 is still valid python?

Reply:
yes, it is still present. We should not use it, but it is there if necessary